cogito/putter.go

```diff
@@ -51,22 +51,40 @@ func (putter *ProdPutter) LoadConfiguration(input []byte, args []string) error {
 		"environment", putter.Request.Env,
 		"args", args)
 
+	// If chat send only is requested, input dir may be empty.
+	// Validate sinks configuration if present, then decide.
+	_, err = putter.Sinks()
+	if err != nil {
+		return fmt.Errorf("put: arguments: %w", err)
+	}
+	sinks := putter.Request.Source.Sinks
+	if len(putter.Request.Params.Sinks) > 0 {
+		sinks = putter.Request.Params.Sinks
+	}
+	sinksSet := sets.From(sinks...)
```

Comment:
Here also I think we can avoid the `Set` suffix and keep the code simpler:

```go
	var sinks *sets.Set[string]
	// if put.params.sinks is not empty it overrides source.params.sinks.
	if len(putter.Request.Params.Sinks) > 0 {
		sinks = sets.From(putter.Request.Params.Sinks...)
	} else {
		sinks = sets.From(putter.Request.Source.Sinks...)
	}
```

Reply:
Done, thanks